TaskWithWritingInFile/TaskWithWritingInFileWay2/TaskWithWritingInFileWay2.cpp

```diff
@@ -0,0 +1,46 @@
+#include <windows.h>
+#include <string>
+#include <iostream>
```

Comment:
It seems nothing from the iostream is used. Please remove the include.

Reply:
Removed